src/components/Gather/Gather.js

```diff
@@ -181,13 +157,7 @@ class Gather extends Component {
 
   setButtonsPhone = () => {
```

Comment:
This entire function should be able to be removed now, since no buttons are pushed